src/kyc_endpoint.py

```diff
@@ -60,6 +60,7 @@ def aplyid_pdf(token=None):
     response.headers.set('Content-Type', 'application/pdf')
     return response
 
+@csrf.exempt
```

Comment:
documentation seems to show the exemption *after* the route (https://flask-wtf.readthedocs.io/en/0.15.x/csrf/#exclude-views-from-protection)